runtime/bcverify/rtverify.c

```diff
@@ -45,7 +45,7 @@ static IDATA matchStack (J9BytecodeVerificationData * verifyData, J9BranchTarget
 static IDATA findAndMatchStack (J9BytecodeVerificationData *verifyData, IDATA targetPC, IDATA currentPC);
 static IDATA verifyExceptions (J9BytecodeVerificationData *verifyData);
 static J9BranchTargetStack * nextStack (J9BytecodeVerificationData *verifyData, UDATA *nextMapIndex, IDATA *nextStackPC);
-static IDATA nextExceptionStart (J9BytecodeVerificationData *verifyData, J9ROMMethod *romMethod, IDATA lastPC);
+static void nextExceptionPC (J9BytecodeVerificationData* verifyData, J9ROMMethod* romMethod, IDATA* exceptionStartPC, IDATA* exceptionEndPC);
```

Comment:
for `*`, the original formatting was correct.
```suggestion
static void nextExceptionPC (J9BytecodeVerificationData*verifyData, J9ROMMethod *romMethod, IDATA* exceptionStartPC, IDATA *exceptionEndPC);
```